docs/oauth/oauth-scoped-access.mdx

```diff
@@ -0,0 +1,125 @@
+
+> [!NOTE]
+> If you would like to know more about the terminology around OAuth, check out our [OAuth terminology guide](/docs/oauth/oauth-overview#key-terminology).
+
+This guide provides step-by-step instructions on how to build OAuth scoped access into a Clerk application.
+
+## Before you start
+
+- You need to [create a Clerk account](https://dashboard.clerk.com/sign-up) or [sign into the Clerk Dashboard](https://dashboard.clerk.com/).
+- You need to create a Clerk Application in the [Clerk Dashboard](https://dashboard.clerk.com/). For more information, see the [setup guide](/docs/quickstarts/setup-clerk).
+
+Let's begin by creating an OAuth application in the Clerk Dashboard.
+
+<Steps>
+  ### Create a Clerk OAuth application
+
+  1. In the Clerk Dashboard, navigate to the [**OAuth Applications**](https://dashboard.clerk.com/last-active?path=user-authentication/oauth-applications) page.
+
+  1. Select the **Add OAuth application** button. A modal will open.
+
+  1. Complete the following fields:
+
+     - `Name` - Helps you identify your application.
+     - `Scopes` - The scopes that you would like to leverage. To find out more about the scopes currently supported by Clerk, check out the [OAuth reference documentation](/docs/oauth/how-clerk-implements-oauth#scopes).
+
+     ![A screenshot of a new OAuth application being created in the Clerk dashboard](/docs/images/oauth/dashboard-create-oauth-application.png)
+
+  1. Select **Add**. A modal will open with your **Client Secret**.
+
+     > [!WARNING]
+     > For security reasons, Clerk does not store your Client Secret and cannot show it to you again, so make sure to copy the secret and store it someplace secure, like an environment variable.
+
+  1. You'll be redirected to your OAuth application's settings page. Under **Application credentials**, you can find your **Client ID**, which will be necessary for to complete the OAuth flow.
+
+  1. In the **Redirect URIs** field, add the redirect URI provided by your client. This is the URL that Clerk will redirect to with an authorization code after the user has authenticated and consented to the access request. The example below uses [http://localhost:3000/oauth\_callback](http://localhost:3000/oauth_callback).
+
+     ![A screenshot of the redirect urls, client id, and client secret within an OAuth application in the Clerk dashboard](/docs/images/oauth/dashboard-oauth-client-details.png)
+
+  The Clerk setup is now complete. If you'd like to know more about other OAuth application configuration settings, refer to the [dedicated reference documentation](/docs/oauth/how-clerk-implements-oauth).
+
+  ### Test the OAuth flow
+
+  In order to test the OAuth flow, you can use the following [OAuth test client](https://github.com/clerk/test-oauth-client) project. It contains a minimal Express.js server that can run through an OAuth flow.
```

Comment:
```suggestion
  If you are aiming to integrate your Clerk application with a third party app that acts as an OAuth client, you can reference their documentation on how to integrate OAuth connections in order to test your OAuth flow. It will likely be necessary to provide the `client_id`, `client_secret`, and `redirect_uri` from the OAuth application we created above.
  
  If you don't have a specific, existing OAuth client in mind, you can use the following [OAuth test client](https://github.com/clerk/test-oauth-client) project. It contains a minimal Express.js server that can run through an OAuth flow. We'll walk through how to use the test client below.
```